distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/ContainerPerSuiteItestSuite.java

```diff
@@ -44,5 +45,6 @@
   TestFanout.class,
   TestOidc.class,
   TestSecurityAuditPlugin.class,
+  TestSplitCatalog.class
```

Comment:
✏️ A lot of effort has been ongoing to decrease DDF's build time. I'm concerned this is the wrong place to iterate on what might be an unstable test, at first. Granted, these only run nightly, so it's not a major deal breaker. But that means flakiness won't be shaken out early.

Reply:
true it will extend the run time, but essential to ensure this new provider is working